### PR TITLE
Stdlib fun

### PR DESCRIPTION
**Pull request by Neitsch**
@kevinye1, see #14

### CODE REVIEW

**Comment by Neitsch** [comment]
@kevinye1 pls fix acos TC

**Review by Neitsch — COMMENTED** [review]
@kevinye1 Please remove magic numbers (see https://github.com/ExtendLang/Extend/pull/14/commits/6ea8cff367f1465f7903fc249fdd65a94e5fed27). Consider writing a helper like setNumber(value_p, double).

**Comment by Neitsch** [comment]
It doesn't look like this built. Please merge with master and commit to trigger the build. @kevinye1 

Also feel free to approve and merge since I opened the PR.

**Review by kevinjye — APPROVED** [review]

**Comment by ishaankolluri** [comment]
Why'd this fail?

**Comment by oracleofnj** [comment]
Looks like test_fabs is failing.

File: testscript.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ for f in $(ls $TESTDIR/$REGRESSION); do
     counterc=$((counterc+1))
     echo "Compiler: PASSED ($f)"
   else
-    echo "Compiler: FAILED ($f)"
+    echo "Compiler: FAILED REGRESSION TEST ($f)"
```
Comment (Neitsch) [inline testscript.sh:62]: Thx

**Comment by oracleofnj** [comment]
I removed test_fabs from inputs_regression directory. Passed, merged, and
deleted branch - please create a new branch for the file IO/plotting
functions you are working on!

**Comment by ishaankolluri** [comment]
Yep! Was planning on adding a new one.

**Pull request merged**